libbeat/docs/breaking.asciidoc

```diff
@@ -9,9 +9,155 @@ changes, but there are breaking changes between major versions (e.g. 5.x to
 
 See the following topics for a description of breaking changes:
 
+* <<breaking-changes-6.3>>
```

Comment:
For completeness, I think we will want to include headings for 6.1 and 6.2 as well since there were also breaking changes in these releases that are currently only documented in the release notes today.

https://www.elastic.co/guide/en/beats/libbeat/6.2/release-notes-6.1.0.html#_breaking_changes_3
https://www.elastic.co/guide/en/beats/libbeat/6.2/release-notes-6.2.0.html#_breaking_changes_2
https://www.elastic.co/guide/en/beats/libbeat/6.2/release-notes-6.2.3.html#_breaking_changes

Reply:
@ppf2 I think the difference with the aboves is that these we features which still were in beta.

Reply:
I'll either add sections or make it clear that this section only covers the main breaking changes. We say that in a couple of places plus provide links to the relnotes, but maybe it's not clear enough.

Reply:
+1, let's clarify.  Are we saying that the breaking changes guide will only cover breaking changes against the GA product?  Or do we mean that it only covers the "main" breaking changes of GA product?  If "main" is the keyword here, let's also confirm that this is consistent with the rest of the Elastic stack.  Afaik, Elasticsearch documents all breaking changes against the GA product, not just the main ones.

If we are only covering breaking changes of the GA product, we can state that for breaking changes against pre-GA product releases, these are documented only in the release notes, etc.. 

thanks!

Reply:
That's a good question, and I'm not sure about the answer. I'd like @tsg or @ruflin to weigh in here.

Reply:
Moving forward we should document here all breaking changes which happen for any GA features. In general no breaking changes in minor should happen, but there are exceptions.

For beta or experimental features, the breaking changes should only be in the changelog.